### PR TITLE
AAP-3543 Update Ansible on Azure networking doc

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Jira link: [AAP-3543](https://issues.redhat.com/browse/AAP-3543)

Update the networking section of the Ansible on Azure doc, as the source information has been modified since publication.

Affects `/titles/aap-on-azure/`

### Versions affected
AAP 2.1

### Supporting documentation:
[Private Network Peering](https://docs.google.com/document/d/1OQY7wn21sAwwqbJPvtGFgI4lVDop7SQW1ss4LGg1Y24/edit) Google doc

### CODE REVIEW

File: downstream/assemblies/aap-on-azure/assembly-aap-azure-connecting-to-aap.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+ifdef::context[:parent-context: {context}]
+
+[id="aap-azure-connecting-to-aap"]
+= Connecting to {PlatformName}
+
+:context: aap-azure-connecting-to-aap
+
+[role="_abstract"]
+Once network peering is performed and your Azure routing configuration is established, you can choose how your team will access {PlatformNameShort} through your Azure network configuration.
```
Comment (ianf77) [inline downstream/assemblies/aap-on-azure/assembly-aap-azure-connecting-to-aap.adoc:9]: When network peering is performed and your Azure routing configuration is established, you can choose how your team accesses {PlatformNameShort} through your Azure network configuration.

File: downstream/assemblies/aap-on-azure/assembly-aap-azure-network-peering.adoc [hunk 2]
```diff
@@ -8,22 +8,35 @@ ifdef::context[:parent-context: {context}]
 // [role="_abstract"]
 // You can use these instructions to install 
 
-Ansible Automation Platform on Azure is deployed into an independent managed resource group with its own Azure virtual network (VNet). 
+{AAPonAzureNameShort} is deployed into an independent managed resource group with its own Azure virtual network (VNet). 
 
-When initially deployed, Ansible Automation Platform on Azure’s VNet can only send requests to external networks through the public internet.
+When initially deployed, {AAPonAzureNameShort}’s VNet can only send requests to external networks through the public internet.
+
+To enable {AAPonAzureNameShort} to access resources in an internet-gapped deployment, when access to resources has to happen over private networks, you must configure Azure network peering between your private virtual networks and {AAPonAzureName}’s managed application VNet.
```
Comment (ianf77) [inline downstream/assemblies/aap-on-azure/assembly-aap-azure-network-peering.adoc:15]: Is internet-gapped the same as air-gapped?

File: downstream/modules/aap-on-azure/proc-azure-nw-private-deploy-vpn.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+[id="proc-azure-nw-private-deploy-vpn"]
+
+= VPN
+
+If your organization requires that many users connect to {PlatformNameShort} over a private connection, or if your organization already uses VPNs or Direct Connections with Azure, then this approach is likely suitable for your needs.
+
+In this case, your on-premises infrastructure is connected to Azure through a Network Application Gateway and has routing rules that can enable access to {PlatformNameShort} to any connected computer on the local network.
+The VNET connected to the Virtual Network Gateway would be connected to other Azure VNETs through network peering with routing rules established to send network traffic for 192.168.0.0/24 to the {PlatformNameShort} VNET.
+
+With this configuration, users would not need to perform any extra configuration; they would be able to access {PlatformNameShort} through the application URLs as if they were using the public access approach.
```
Comment (ianf77) [inline downstream/modules/aap-on-azure/proc-azure-nw-private-deploy-vpn.adoc:17]: users would not need to -> need not
they can access {PlatformNameShort}

File: downstream/modules/aap-on-azure/proc-azure-nw-private-deploy.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+[id="proc-azure-nw-private-deploy"]
+
+= Private deployments
+
+If you selected private access when deploying {AAPonAzureNameShort},
+then the DNS record issued to the {AAPonAzureName} application points to a private Class C network address: 192.168.0.68.
+This address is not accessible from external sources, even from other Azure devices: you must configure access.
+
+The configuration and access method that you choose to connect to {AAPonAzureNameShort} will depend on how your organization manages Azure infrastructure.
```
Comment (ianf77) [inline downstream/modules/aap-on-azure/proc-azure-nw-private-deploy.adoc:16]: depends on how your organization

File: downstream/modules/aap-on-azure/proc-azure-nw-private-deploy.adoc [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+Your Azure administrators will need to determine the right model for your organization and configure the setup for you.
```
Comment (ianf77) [inline downstream/modules/aap-on-azure/proc-azure-nw-private-deploy.adoc:17]: administrators must determine

File: downstream/modules/aap-on-azure/proc-azure-resource-quotas.adoc [hunk 6]
```diff
@@ -30,23 +30,23 @@ Ensure that you have selected a single region. Viewing all regions at once will
 
 The `StandardCore` limit is a compute metric for the resources that are temporarily consumed when deploying the managed application.
 
-It is possible that the Ansible Automation Platform for Azure will deploy without hitting the `StandardCore` limit.
+It is possible that the {AAPonAzureNameShort} will deploy without hitting the `StandardCore` limit.
```
Comment (ianf77) [inline downstream/modules/aap-on-azure/proc-azure-resource-quotas.adoc:33]: deploys without

**Review by ianf77 — COMMENTED** [review]
A few comments

File: downstream/assemblies/aap-on-azure/assembly-aap-azure-network-peering.adoc (hunk 2 above)
Comment (ariordan-redhat) [reply downstream/assemblies/aap-on-azure/assembly-aap-azure-network-peering.adoc:15]: @ianf77 
It's not the same as air-gapped. Scott says it means that external requests via public internet are not permitted, but they are permitted over private networking.

**Pull request merged**